api/net/Areas/Subscriber/Controllers/ReportInstanceController.cs

```diff
@@ -209,6 +210,28 @@ public async Task<IActionResult> SendToAsync(int id, string to)
         return new JsonResult(new ReportInstanceModel(instance));
     }
 
+    /// <summary>
+    /// Generate an Excel document report.
+    /// </summary>
+    /// <param name="instanceId"></param>
+    /// <returns></returns>
+    [HttpPost("{instanceId}/export")]
```

Comment:
not a big deal, but wondering whether semantically speaking should this be a GET rather than a POST?

Reply:
done ! :)